changelog/58715.changed

```diff
@@ -0,0 +1,2 @@
+Changed `check_boto_reqs()` rely on the specified `boto_ver` or `boto3_ver` to decide which versions of boto to test.
+Changed all boto/boto3 code to explicitly specify which boto version they need.
```

Comment:
IIUC towncrier might make this a bit wonky - you may want to try [generating the changelog](https://docs.saltstack.com/en/master/topics/development/changelog.html#how-to-generate-the-changelog) yourself to see if two lines look reasonable.

The alternative would just to make it one long line, and let towncrier do any wrapping for you.

Reply:
Yah, I noticed this odd formatting as well and wasn't entirely certain if there was any way around it.

Reply:
Reworded as a single line.